airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -301,7 +301,12 @@ def _read_grouped_logs(self):
         return True
 
     def _read(
-        self, ti: TaskInstance, try_number: int, metadata: dict | None = None
+        self,
+        ti: TaskInstance,
+        try_number: int,
+        metadata: dict | None = None,
+        offset: int = 0,
```

Comment:
I would just leave the `es_task_handler` alone right now

Reply:
This is needed to match the _read signature that it overrides

Reply:
It has to be opt in, so if a user upgrades to the version of Airflow that supports task log pagination, it mustn't break. Unless the logic is actually implemented for logs served from ES, I think it'd break, so by default, there should be no pagination.